### DISCUSSION TITLE
[Feature Request] Timer: Add support for longer than an hour

### DISCUSSION THREAD

**Discussion by kop316** [InfiniTimeOrg/InfiniTime#1678] [Ideas]
### Verification

- [X] I searched for similar issues and found none was relevant.

### Introduce the issue

Hello! I noticed that the timer can only go up to `59:59`. It would be nice if I make a timer for longer than that.

### Preferred solution

Possibly, add a third column for hours?

**Comment by MyNameIsTroll** [comment]
Iwas gonna propose the same thing, thank you for opening a discussion.
I find it usefull to be able to put a timer when i go to bed in irregular hours, for example to sleep 7 hours or when i launch my washing machine to get a reminder to go out the linen in 2h30 
:)

**Comment by Boteium** [comment]
I second that

**Comment by Henry-Hiles** [comment]
This would be great, its quite annoying right now, thanks.
